src/generator-filters.cs

```diff
@@ -357,7 +357,7 @@ void GenerateFilterSetter (string propertyType, string propertyName)
 			print ("}");
 			break;
 		default:
-			throw new BindingException (1018, true, "Unimplemented CoreImage property type {0}", propertyType);
+			throw new BindingException (1075, true, propertyType);
```

Comment:
Same diff error number.

Reply:
The error code 1018 is being used with a different error message in other parts of the generator.  The message for 1018 should be: `No [Export] attribute on property *.*`.  I also changed the error codes for other existing exceptions  because they were using the same error code for different messages.

Reply:
All of you other comments about diff error codes have the same answer so I've marked them as resolved.

Reply:
All of the comments about error numbers have the same reason so I've marked them as resolved.